src/rez/plugin_managers.py

```diff
@@ -201,9 +176,55 @@ def load_plugins(self):
                         traceback.print_exc(file=out)
                         print_debug(out.getvalue())
 
-            # load config
-            data, _ = _load_config_from_filepaths([os.path.join(path, "rezconfig")])
-            deep_update(self.config_data, data)
+    def load_plugins_from_entry_points(self):
+        if sys.version_info.minor >= 8:
+            from importlib.metadata import entry_points
+        else:
+            from importlib_metadata import entry_points
+
+        discovered_plugins = entry_points(group='rez.plugins')
+        for plugin in discovered_plugins:
+            plugin = plugin.load()
+            plugin_name = plugin.__name__.split('.')[-1]
+            plugin_path = os.path.dirname(plugin.__file__)
+            self.register_plugin_module(plugin_name, plugin, plugin_path)
+            self.load_config_from_plugin(plugin)
```

Comment:
We should put this code under a try/except block and add logs similar to logs found in `load_plugins_from_namespace`.